pkg/vm/engine/disttae/txn.go

```diff
@@ -361,16 +400,90 @@ func blockUnmarshal(data []byte) BlockMeta {
 	return BlockMeta{}
 }
 
-/*
 // write a block to s3
-func blockWrite(ctx context.Context, blkInfo BlockMeta, bat *batch.Batch) error {
-	//TODO
-	return nil
+func blockWrite(ctx context.Context, blkInfo BlockMeta, bat *batch.Batch, fs fileservice.FileService) ([]objectio.BlockObject, error) {
+	// 1. check columns length, check types
+	if len(blkInfo.columns) != len(bat.Vecs) {
```

Comment:
block may be a delete block

Reply:
unsupported in objectio now.
i will add these logic when the interface ready.